api/openapi.yaml

```diff
@@ -1256,172 +1399,137 @@ components:
-              type: string
-              example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
-            createdAt:
-              description: |
-                The time the feature was created.
-              type: string
-              format: date-time
-              example: "2023-01-01T00:00:00Z"
-            updatedAt:
-              description: |
-                The time the feature was last updated.
-              type: string
-              format: date-time
-              example: "2023-01-01T00:00:00Z"
-    CreateLedger:
-      x-go-type-import:
-        path: github.com/openmeterio/openmeter/internal/credit
-      x-go-type: credit.Ledger
+        - $ref: "#/components/schemas/FeatureCreateInputs"
+        - $ref: "#/components/schemas/SharedMetaFields"
+    EntitlementCreateInputs:
       type: object
       description: |
-        A ledger represented in open meter. A ledger must be assigned to a single
-        subject.
+        Entitles a subject to use a feature.
       required:
-        - subject
+        - featureId
+      additionalProperties: false
       properties:
-        subject:
+        featureId:
           description: |
-            The metering subject this ledger used to track grants for.
-          type: string
-        metadata:
-          type: object
-          additionalProperties:
-            type: string
-          example:
-            stripePaymentId: "pi_4OrAkhLvyihio9p51h9iiFnB"
-    LedgerAlreadyExistsProblem:
-      description: Ledger Exists
-      x-go-type-import:
-        path: github.com/openmeterio/openmeter/internal/credit
-      x-go-type: credit.LedgerAlreadyExistsProblemResponse
-      type: object
-      # TODO: until https://github.com/deepmap/oapi-codegen/pull/1610 is not merged, we
-      # need to copy the Problem schema here
-      #
-      # Afterwards the schema should be this
-      #allOf:
-      #- $ref: "#/components/schemas/Problem"
-      #- type: object
-      #  properties:
-      #    conflictingEntity:
-      #      $ref: "#/components/schemas/Ledger"
-      properties:
-        type:
-          type: string
-          format: uri
-          description: Type contains a URI that identifies the problem type.
-          example: "urn:problem-type:bad-request"
-        title:
-          type: string
-          description: A a short, human-readable summary of the problem type.
-          example: Bad Request
-        status:
-          type: integer
-          format: int32
-          description: The HTTP status code generated by the origin server for this occurrence of the problem.
-          minimum: 400
-          maximum: 599
-          example: 400
-        detail:
+            The feature to grant.
           type: string
-          description: A human-readable explanation specific to this occurrence of the problem.
-          example: "body must be a JSON object"
-        instance:
+          example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
+        measureUsageFrom:
```

Comment:
As it's related to period I'd call it `periodAnchor`

Reply:
The idea here was actually different, it was so we can have a "different first period" (longer shorter)